osu.Game.Rulesets.Catch/Edit/DrawableCatchEditorRuleset.cs

```diff
@@ -13,11 +14,24 @@ namespace osu.Game.Rulesets.Catch.Edit
 {
     public class DrawableCatchEditorRuleset : DrawableCatchRuleset
     {
+        public readonly BindableDouble TimeRangeMultiplier = new BindableDouble(1);
```

Comment:
I think xmldoc should be added to this at very least, it's a bit nondescript as it stands (especially since this generally wouldn't be used during normal gameplay scenarios).

Reply:
I guess I'll survive.